### PR TITLE
Remove final two switch(sqlType) statements from `SSTableModel`.

### PR DESCRIPTION
**Pull request by errael**
Remove final two switch(sqlType) statements from `SSTableModel`.
Add `getColumnObject()` to `RowSetOps`.

### CODE REVIEW

File: swingset/src/main/java/com/nqadmin/swingset/datasources/RowSetOps.java [hunk 1]
```diff
@@ -894,4 +894,72 @@ public static void updateColumnObject2(RowSet _rowSet, Object _value, int _colum
...
+	 * Fetch and return object for specified rowset's specified column;
+	 * type is determined by column's JDBCType.
+	 * @param rowset source rowset
+	 * @param _column source column
+	 * @return 
+	 * @throws SQLException
+	 */
+	public static Object getColumnObject(RowSet rowset, int _column) throws SQLException
+	{
+		if (Boolean.TRUE)
```
Comment (bpangburn) [inline swingset/src/main/java/com/nqadmin/swingset/datasources/RowSetOps.java]: Won't `if (Boolean.TRUE)` always resolve to true?

Comment (errael) [reply swingset/src/main/java/com/nqadmin/swingset/datasources/RowSetOps.java]: Right. I think it's OK to replace the 40+ lines by a single line. But that's based on my experience with `H2`; looking at specs and h2 code. I'm not that familiar with other JDBC drivers. I have learned through reading that they can vary greatly. In fact `SQLFeatureNotSupportedException` is listed for almost every method in ResultSet javadoc. So this change leaves both the original and one liner as something that can be easily switched.

**Comment by errael** [comment]
Changes in comments: see https://github.com/bpangburn/swingset/compare/65855b120975e109aea8211adcc2c5dc80b3cb2d..5d3ab2b4611e76ebf50d61ffc98921a63ee599fc .

**Review by bpangburn — APPROVED** [review]
Looks good.

**Pull request merged**